src/CSV_Parser.php

```diff
@@ -292,6 +331,29 @@ public function parse($headers = true) {
 	}
 
 	/**
+	 * Convert character encoding of data
+	 * @throws Jabran\Exception\InvalidEncodingException
+	 * @return self
+	 */
+	public function convertEncoding() {
```

Comment:
Also does this method check if an encoding type is already set or still NULL (default)? It might also add to my earlier comment about setting a default encoding.